package.json

```diff
@@ -53,5 +53,8 @@
   "bugs": {
     "url": "https://github.com/laws-africa/indigo/issues"
   },
-  "homepage": "https://github.com/laws-africa/indigo#readme"
+  "homepage": "https://github.com/laws-africa/indigo#readme",
+  "dependencies": {
+    "htmx": "^0.0.2"
+  }
```

Comment:
```suggestion
  "homepage": "https://github.com/laws-africa/indigo#readme"
```

Reply:
htmx.org is the real dependency and is above

Reply:
ah thanks, I reverted all this